Zend/zend_vm_gen.php

```diff
@@ -519,67 +519,56 @@ function gen_code($f, $spec, $kind, $export, $code, $op1, $op2, $name) {
 	// Updating code according to selected threading model
 	switch($kind) {
 		case ZEND_VM_KIND_CALL:
-			$code = preg_replace_callback(
```

Comment:
I suggest not update this file, just test it in PHPT is enough, since we have to need PHP7 to generate new zend_vm*, that seems not necessary and make things complex.